src/tasks/AutoHeistTask.py

```
@@ -52,7 +52,10 @@ def _inst_gap():
 INST = "<br>".join(
     [
         _inst_line("📍 步骤起点：站在可互动小吱的位置开始", "#FF5555", bold=True),
-        _inst_line("⚙️ 镜头设置：控制 ➔ 移动镜头修正 ➔ 禁用", "#FF5555", bold=True),
+        _inst_line("⚙️ 镜头设置", "#FF5555", bold=True),
+        _inst_line("└─ 控制 ➔ 摄像机设置", "#FE821D", bold=True, indent=1),
+        _inst_line("├─ 移动镜头修正：禁用", "#FE821D", bold=True, indent=2),
```

Ruff (0.15.14)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)